src/posix.mak

```diff
@@ -370,12 +370,13 @@ FORCE: ;
 # Generate the man pages
 ################################################################################
 
-DMD_MAN_PAGE = $(GENERATED)/docs/man1/dmd.1
+DMD_MAN_PAGE = $(GENERATED)/docs/man/man1/dmd.1
```

Comment:
This variable is now unused.

Reply:
Actually it's still used in the "install" target a few lines down.